apim/3.x/templates/gateway/gateway-configmap.yaml

```diff
@@ -360,6 +360,16 @@ data:
         enabled: true
         ws:
           discovery: true
+          sendEventsOnHttp: {{ .Values.alerts.sendEventsOnHttp | default true }}
```

Comment:
I would prefer a block of code to handle this part. 
It's more flexible.
WDYT?